packages/utils-telemetry/src/telemetry-api.ts

```diff
@@ -3,30 +3,23 @@ type Properties = { [key: string]: string };
 
 type TelemetryItem = {
     data: {
-        baseData: {
-            measurements: Measurements;
-            name: string;
-            properties: Properties;
-            ver?: number;
-        };
-        baseType: string;
+        measurements: Measurements;
+        name: string;
+        properties: Properties;
     };
-    iKey: string;
-    name: string;
+    type: string;
     time: string;
 };
 
-const appInsightsEndpoint = 'https://dc.services.visualstudio.com/v2/track';
+const telemetryEndPoint = 'https://webhint-telemetry.azurewebsites.net/api/log';
```

Comment:
@hxlnt just want confirmation this is what we want: to use our Azure Function to report the telemetry to app insights so we can clean up any extra information that we do not want to store.
Is that correct?

Reply:
SGTM.